Makefile

```diff
@@ -33,10 +33,10 @@ export GOPROXY
 export GO111MODULE=on
 
 # Go version
-GOLANG_VERSION := 1.22.11
+GOLANG_VERSION := 1.23.8
 
 # Kubebuilder
-export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.31.0
+export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.32.2
```

Comment:
All these `32.2` need to be `32.0` as there is not a Kubebuilder tools published version for k8s > 1.32.0